catalogue/bpf/tests/LB+fcas-addr-once+once-scas.litmus

```diff
@@ -0,0 +1,25 @@
+BPF LB+fcas-data-once+once-scas
```

Comment:
```suggestion
BPF LB+fcas-addr-once+once-scas
```

Reply:
Notice: I found this inconstency (and the next one) with the `mdiag7` tools:
```
% mdiag7 *.litmus
Error: name LB+fcas-ctrl-once+once-scas has different hashes
 LB+fcas-ctrl-once+once-scas.litmus
 LB+fcas-ctrlcvg-once+once-scas.litmus
Error: name LB+fcas-data-once+once-scas has different hashes
 LB+fcas-data-once+once-scas.litmus
 LB+fcas-addr-once+once-scas.litmus
Input: 34
By name: 32
By hash: 34
```

Reply:
Thanks, I fixed these and reran `mdiag7` to verify it is working.